src/schema/v2/collection.ts

```diff
@@ -1,114 +0,0 @@
-import { pageable } from "relay-cursor-paging"
-import { connectionFromArraySlice } from "graphql-relay"
-import cached from "./fields/cached"
-import CollectionSorts from "./sorts/collection_sorts"
-import { artworkConnection } from "./artwork"
-import {
-  CatchCollectionNotFoundException,
-  convertConnectionArgsToGravityArgs,
-} from "lib/helpers"
-import { NodeInterface, SlugAndInternalIDFields } from "./object_identification"
-import {
-  GraphQLObjectType,
-  GraphQLString,
-  GraphQLNonNull,
-  GraphQLBoolean,
-  GraphQLFieldResolver,
-  GraphQLFieldConfig,
-} from "graphql"
-import { ResolverContext } from "types/graphql"
-import { includesFieldsOtherThanSelectionSet } from "lib/hasFieldSelection"
-
-// Note to developers working on collections, the staging server does not get a copy
-// of all artwork saves, so you will need to add some each week in order to have data
-// to work with.
-
-// TODO: Is the root collection field really used and do we still need this?
-export const CollectionType = new GraphQLObjectType<any, ResolverContext>({
```

Comment:
This original implementation was [initially added in 2017](https://github.com/artsy/metaphysics/pull/620) and then [pruned in 2019 as part of the MP v2 effort](https://github.com/artsy/metaphysics/pull/1920). 

(The old v1 api itself was [dropped in its entirety a few months ago](https://github.com/artsy/metaphysics/pull/4262), though this unused code was left in place.)

I opted to drop this old, orphaned implementation and start fresh with a schema design that more closely reflects our product need and leans less on some of the tricky patterns seen here. I hope we don't end up re-inventing these wheels. Let me know if you have any concerns about this.